strings/strings.json

```
@@ -7606,5 +7606,9 @@
   "OrganizationWillResume": {
     "en": "Your organization is resuming and will be available shortly.",
     "fr": "Votre organisation redeviendra active et sera bientôt disponible."
+  },
+  "UpdatingResults": {
+    "en": "Updating results",
+    "fr": "Mise à jour des résultats"
```

I don't feel like this is clear. Maybe this?
```suggestion
    "fr": "Chargement des résultats"
```